lib/processors/jsdoc/lib/ui5/plugin.js

```diff
@@ -803,8 +806,8 @@ function resolveFullyQuantifiedName(node, type) {
 
 		if (potentialChunk && potentialChunk.renamed) {
 			originalName = originalName.replace(
-				potentialChunk.renamed + ".",
-				potentialChunk.original + "."
+				new RegExp("(^|.)" + potentialChunk.renamed + "($|.)"),
```

Comment:
Is there a need to escape characters for the RegExp?

Reply:
Replaced the Regex with a string comparison function